taskt/UI/Forms/frmScriptBuilder.Designer.cs

```diff
@@ -52,7 +52,7 @@ private void InitializeComponent()
             this.recordToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
             this.scheduleToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
             this.runToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
```

Comment:
Can you please cleanup `System.Windows.Forms` namespace references through out the file?

Reply:
So we are not going to update auto-generated code (as decided).